gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/GenerateDataTemplateTask.java

```diff
@@ -127,8 +127,20 @@ public void setEnableArgFile(boolean enable)
     _enableArgFile = enable;
   }
 
-  @Input
+  /**
+   * @deprecated by {@link #isGenerateFieldMask()} because Gradle 7 requires
+   *     input and output properties to be annotated on getters, which have a
+   *     prefix of "is" or "get".
+   */
+  @Deprecated
```

Comment:
Perhaps also mark `@Internal`?

Reply:
Apparently, Gradle 7 complains that the getter does not have the right naming convention, even for `@Internal`.

Reply:
ok!